tests/CLI/modules/image_tests.py

```diff
@@ -53,3 +53,15 @@ def test_datacenter_remove(self):
     def test_datacenter_remove_fails(self):
         result = self.run_command(['image', 'datacenter', '100', '--remove'])
         self.assertEqual(2, result.exit_code)
+
+    def test_deny_share(self):
```

Comment:
is possible add an assert to validate the image does not show in the image list

Reply:
Done